schema.sql

```diff
@@ -11,3 +11,17 @@ CREATE TABLE animals(
 );
 
 ALTER TABLE animals ADD species VARCHAR;
+
+CREATE TABLE owners(
+   id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+   full_name VARCHAR NOT NULL,
+   age INT);
+
+CREATE TABLE species(
+   id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+   name VARCHAR NOT NULL);
+
+ALTER TABLE animals ADD PRIMARY KEY(id);
```

Comment:
- [ ] According to the requirements below, you are missing the highlighted modification to your animal table. Please include it
<img width="647" alt="Screenshot 2023-10-06 at 21 05 07" src="https://github.com/BiancaMe/vet-db/assets/64774936/04957775-04f0-4a2a-8167-7154b498143f">